test/end-to-end/core.sh

```diff
@@ -18,55 +19,55 @@ TEST_ASSETS="${TEST_ASSETS:-false}"
 function wait_for_app() {
   echo "[INFO] Waiting for app in namespace $1"
   echo "[INFO] Waiting for database pod to start"
-  wait_for_command "oc get -n $1 pods -l name=database | grep -i Running" $((60*TIME_SEC))
-  oc logs dc/database -n $1 --follow
+  os::cmd::try_until_text "oc get -n $1 pods -l name=database" 'Running'
+  os::cmd::expect_success 'oc logs dc/database -n $1 --follow' # TODO(skuznets): determine why we use --follow here
```

Comment:
@liggitt Do we want to grab the build logs at a later point or do we need `--follow` here for some reason?